frontend/src/app/main/ui/dashboard/sidebar.cljs

```diff
@@ -939,9 +954,21 @@
 
         handle-set-profile
         (mf/use-fn
-         #(on-click :settings-profile %))]
+         #(on-click :settings-profile %))
+
+        on-power-up-click
+        (mf/use-fn
+         (fn []
+           (dom/open-new-window "https://penpot.app/pricing")))]
 
     [:*
+     [:div {:class (stl/css :upgrade-plan-section)
+            :on-click on-power-up-click}
```

Comment:
This looks like it should be a `<button>` tag.

Reply:
done